zeitschrift-fur-allgemeinmedizin.csl

```diff
@@ -14,12 +14,13 @@
     <category field="medicine"/>
     <issn>1433-6251</issn>
     <eissn>1439-9229</eissn>
-    <updated>2017-07-25T14:09:51+00:00</updated>
+    <updated>2017-07-26T21:25:26+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="de">
     <terms>
       <term name="accessed">letzter Zugriff am</term>
+      <term name="et-al">et al</term>
```

Comment:
Even then the term should have a period. (term definitions should be correct independent of the context in which they're used, and CSL processors generally suppress duplicated punctuation to deal with e.g. extra periods)